spec/requests/security_activity_spec.rb

```diff
@@ -154,13 +167,24 @@
     expect_event SecurityActivity::EMAIL_CHANGED, notes: "to #{user.email}"
   end
 
-  def expect_event(event, application: nil, factor: nil, notes: nil, analytics: nil)
-    events = user.security_activities.of_type(event)
-    events = events.where(oauth_application_id: application.id) if application
-    events = events.where(factor: factor) if factor
-    events = events.where(notes: notes) if notes
-    events = events.where(analytics: analytics) if analytics
+  def expect_event(event, **options)
```

Comment:
If you make this `def expect_event(event, options={})`, then all the `{ ... }` changes above are unnecessary

Reply:
Amended that, see [here](https://github.com/alphagov/govuk-account-manager-prototype/pull/571/files/1d56ff03828272851efcfd7af0d77184c6b990fc#r553832047)

I figured I'd keep the curlies above, they're a very minor syntax addition, and should make it easier if we had more tests that we wanted to add multiple options keys?